manager/controllers/app/moduleinstance.go

```diff
@@ -134,18 +135,20 @@ func (m *ModuleManager) selectReadModule(item modules.DataInfo, appContext *app.
 	m.Log.Info("Select read path for " + item.Context.DataSetID)
 
 	// Read policies for data that is processed in the workload geography
-	var readActions []*pb.EnforcementAction
+	var readActions []*openapiclientmodels.ResultItem
 	var err error
-	readActions, err = LookupPolicyDecisions(item.Context.DataSetID, m.PolicyManager, appContext,
-		&pb.AccessOperation{Type: pb.AccessOperation_READ, Destination: m.WorkloadGeography})
+	reqAction := new(openapiclientmodels.PolicyManagerRequestAction)
```

Comment:
We typically don't use `new` in our codebase. Please replace with something like:
`reqAction := openapiclientmodels.PolicyManagerRequestAction{}`

Reply:
fixed. Thanks.